src/webgpu/api/operation/buffers/map.spec.ts

```diff
@@ -367,12 +367,19 @@ g.test('mappedAtCreation,mapState')
   .desc('Test that exposed map state of buffer created with mappedAtCreation has expected values.')
   .params(u =>
     u
-      .combine('validationError', [false, true])
+      .combine('usageType', ['invalid', 'MAP_READ', 'MAP_WRITE'])
```

Comment:
nit: since MAP_READ doesn't map directly to MAP_READ (but to MAP_READ|COPY_DST), I'd rename these to simply 'read' and 'write' to avoid that false implication.

Reply:
Thanks, updated.